featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -629,15 +669,63 @@ def last_n(df):
             # Apply the non-aggregable functions generate a new dataframe, and merge
             # it with the existing one
             if len(to_apply):
-                wrap = agg_wrapper(to_apply, self.time_last)
-                # groupby_var can be both the name of the index and a column,
-                # to silence pandas warning about ambiguity we explicitly pass
-                # the column (in actuality grouping by both index and group would
-                # work)
-                to_merge = base_frame.groupby(base_frame[groupby_var], observed=True, sort=False).apply(wrap)
-                frame = pd.merge(left=frame, right=to_merge,
-                                 left_index=True,
-                                 right_index=True, how='left')
+                if isinstance(base_frame, dd.DataFrame):
+                    aggregations = {}
```

Comment:
it looks like the body of this if statement isn't being tested

Reply:
I believe we don't have any multi-input/output primitives that are currently supported in Dask, which is what this block deals with.